packages/nvidia_nat_fastmcp/tests/server/test_tool_converter.py

```
@@ -0,0 +1,151 @@
+            "cik-A": (str, "default"),
+        })  # type: ignore[call-overload]
+
+        mock_sm = _mock_session_manager(result_value="ok")
+        wrapper = create_function_wrapper("tool", mock_sm, schema)
+
+        result = await wrapper(**{"from_": "2024-01-01", "cik_A": "ABC"})
+
+        assert result == "ok"
+        mock_sm.run.assert_called_once()
+        payload = mock_sm.run.call_args[0][0]
+        assert getattr(payload, "from") == "2024-01-01"  # noqa: W601
```

Ruff (0.15.17)

[warning] Invalid rule code in `# noqa`: W601

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)